test/packages/benchmarks/rally_benchmark/data_stream/testds/agent/stream/filestream.yml.hbs

```diff
@@ -0,0 +1,4 @@
+paths:
```

Comment:
As discussed above, seems we don't need all the input configs. Even though it does not matter for the test run, lets keep the sample / test package as minimal as possible and remove everything related to inputs.